### PR TITLE
Feat: Added category page

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
category page created.

### CODE REVIEW

**Review by atul-gairola — APPROVED** [review]
Seems good!

**Review by partha8 — APPROVED** [review]
Looks good, good use of semantic html and nice UI

File: css/category.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@import url(./style.css);
+
+main .quiz-categories{
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
```
Comment (pradipgharge) [inline css/category.css:9]: You have defined justify-content two times for same class

**Review by pradipgharge — APPROVED** [review]
looks good to merge

**Pull request merged**